src/clinical/donor-repo.ts

```diff
@@ -381,4 +397,19 @@ SampleSchema.plugin(AutoIncrement, {
   start_seq: process.env.SAMPLE_ID_SEED || 610000,
 });
 
+FollowUpSchema.plugin(AutoIncrement, {
+  inc_field: 'followUpId',
+  start_seq: 1,
+});
+
+PrimaryDiagnosisSchema.plugin(AutoIncrement, {
+  inc_field: 'primaryDiagnosisId',
+  start_seq: 1,
+});
+
+TreatmentSchema.plugin(AutoIncrement, {
+  inc_field: 'treatmentId',
+  start_seq: 1,
+});
+
```

Comment:
Looks like we are not adding these ids for the therapies..

Reply:
yeah they don't consider therapies as entities they are just extra treatment data for some treatment types

Reply:
ok , yea I see it in the mock up